InnerEye/ML/model_training.py

```diff
@@ -53,11 +53,20 @@ def model_train(config: ModelConfigBase,
     :param config: The arguments which specify all required information.
     :param checkpoint_handler: Checkpoint handler object to find checkpoint paths for model initialization
     """
-    checkpoint_callback = ModelCheckpoint(
-        dirpath=str(config.checkpoint_folder),
-        filename='best_val_loss_checkpoint',
-        monitor=f"{VALIDATION_PREFIX}{MetricType.LOSS.value}",
-        save_last=True)
+    # Get the path to the checkpoint to recover from
+    checkpoint_path = checkpoint_handler.get_recovery_path_train()
+
+    last_checkpoint_callback = ModelCheckpoint(dirpath=str(config.checkpoint_folder),
+                                               filename='best_val_loss_checkpoint',
+                                               monitor=f"{VALIDATION_PREFIX}{MetricType.LOSS.value}",
+                                               save_last=True)
```

Comment:
Maybe we should break it down even further: One checkpoint handler that only saves the last epoch's checkpoint, and one that monitors validation loss.